lib/ansible/modules/service.py

```diff
@@ -1010,8 +1016,21 @@ def get_service_tools(self):
 
         self.sysrc_cmd = self.module.get_bin_path('sysrc')
 
+    def run_service_cmd(self, status=None):
+        if not status:
+            return (None, '', '')
```

Comment:
Perhaps, status shouldn't have a default or accept `None` given that Matt pointed out that an empty string or `None` is never passed. We could also require it to be passed as a kwarg since you're already doing so when invoking.

The typing could look like
```suggestion
    def run_service_cmd(self, *, status: str) -> tuple[int, str, str]:
```

(although, MyPy will probably be unhappy with the last return line as it calls an untyped function, meaning it'd be treated as `typing.Any`; you'll probably have to wrap it into a `typing.cast()` for now, to make that go away)

P.S. Is it possible for `self.execute_command()` to stick a `None` in the first tuple element? Probably not, since it's coming from `Popen` after a `wait()` so it should be always set to non-`None` by the time it's accessed. I'm not suggesting `| None` for the following two, since the existing code calling it relies on it being a `str` unconditionally, everywhere.